municipal-services/bpa-services/src/main/java/org/egov/bpa/service/BPAService.java

```
@@ -590,6 +609,44 @@ public BPA update(BPARequest bpaRequest) {
 
     }
 
+	private String getOccupancyCertificateNo(BPARequest bpaRequest) {
+
+		List<IdResponse> idResponses = idGenRepository
+				.getId(bpaRequest.getRequestInfo(), bpaRequest.getBPA().getTenantId(),
+						config.getOccupancyCertificateIdgenName(), config.getOccupancyCertificateIdgenFormat(), 1)
+				.getIdResponses();
+
+		if (idResponses != null && !idResponses.isEmpty()) {
+			return idResponses.get(0).getId();
+		}
+		return "";
```

instead of returning empty string throw the exception from here